tests/routes/search_test.py

```diff
@@ -206,3 +206,19 @@ def test_search_escaped_equal_criteria(self):
         msg = "u'imCool\\\\=notCoolDude' is not of type u'array', "
         msg += "u'imCool\\\\=notCoolDude' does not match u'(?<!\\\\\\\\)='"
         self.search_with_bad_criteria({"search": "imCool\=notCoolDude"}, msg)
+
+    def test_path_search_doesnt_match_artifact(self):
+        self.add_metadata("/test/artifact/test/alsoATest")
+        self.add_metadata("/test/artifact/test_lol")
+        self.search_wrapper.refresh_index()
+        self.route_tester \
+            .search() \
+            .route_params(bucket_name="test", path="/test") \
+            .expect(204, headers={
+                "Link": [
+                    "</test/artifact/test/alsoATest>; rel=\"item\"; title=\"artifact\""
+                ]
+            }) \
+            .post({
+                "search": "artifactName=*"
+            }, headers=self.auth)
```

Comment:
Put "headers" on its own line. Also put the parenthesis on its own line and outdent it.

See the bottom of https://tests-always-included.github.io/style-guide/lang/py/